composition/test/composition_srv.regex

```diff
@@ -0,0 +1 @@
+Incoming request: \[a: \d+, b: \d+\]
```

Comment:
This should not check for a request but rather on the client side for a response.

Reply:
Somehow this commit has been lost in the finally merged PR. I will add it as part of ros2/build_cop#80.